### DISCUSSION TITLE
Audio downmixing on multichannel sources

### DISCUSSION THREAD

**Discussion by Katzenwerfer** [obsproject/obs-studio#8932] [Development Questions] [answered]
I was curious if there is any downmixing being down when trying to capture a surround source through the application capture feature.
So far, I've been unable to figure this out since games mix audio in some odd ways, but to my knowledge any extra channels are just skipped entirely unless you have surround audio enabled.

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
